generate_fragmented_sharded_collection.py

```diff
@@ -186,7 +186,13 @@ def gen_chunks(num_chunks):
 
     def generate_inserts(chunks_subset):
         chunk_size = random.randint(args.chunk_size_min, args.chunk_size_max)
+        doc_size_deviation = args.doc_size_deviation
         doc_size = args.doc_size
+
+        if doc_size_deviation > 0.0:
+            deviation = doc_size * doc_size_deviation
+            doc_size = random.randint(doc_size - deviation, doc_size + deviation)
```

Comment:
What happens if `doc_size - deviation` is negative? Should we add a check to prevent that?

Reply:
The only way for it to be zero would be when `doc size < 1kb`, I don't think it can ever get negative if the arguments are correct.

Added some asserts to be sure